addons/mail/static/src/components/partner_im_status_icon/partner_im_status_icon.scss

```diff
@@ -1,49 +1,12 @@
 // ------------------------------------------------------------------
 // Layout
 // ------------------------------------------------------------------
-
```

Comment:
Why remove this space? :) I think you should put it back to keep the right structure.

Reply:
Updated 👍